<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1455</width>
    <height>930</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ZhaoCong's Flight Planning Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="main_tabs">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="route_info_tab">
       <attribute name="title">
        <string>Route Information</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="start_airport_label">
              <property name="text">
               <string>Start Airport ID:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="start_airport_lineedit"/>
            </item>
            <item>
             <widget class="QLabel" name="end_airport_label">
              <property name="text">
               <string>End Airport ID:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="end_airport_lineedit"/>
            </item>
            <item>
             <widget class="QPushButton" name="get_route_info_button">
              <property name="text">
               <string>Get Route Info</string>
              </property>
              <property name="autoDefault">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableView" name="route_info_table">
            <property name="contextMenuPolicy">
             <enum>Qt::ActionsContextMenu</enum>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::AdjustIgnored</enum>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="verticalScrollMode">
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
            <property name="horizontalScrollMode">
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="airport_info_tab">
       <attribute name="title">
        <string>Airport Information</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <layout class="QGridLayout" name="gridLayout_8">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item row="10" column="0">
           <widget class="CustomQTableView" name="runway_info_table">
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="verticalScrollMode">
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
            <property name="horizontalScrollMode">
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
           </widget>
          </item>
          <item row="12" column="0" colspan="2">
           <widget class="QTextBrowser" name="atis_display"/>
          </item>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="enter_airport_id_label">
              <property name="text">
               <string>Enter Airport ID:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="airport_id_lineedit"/>
            </item>
            <item>
             <widget class="QPushButton" name="get_airport_info_button">
              <property name="text">
               <string>Get Airport Info</string>
              </property>
              <property name="autoDefault">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QTableView" name="airport_info_table">
            <property name="contextMenuPolicy">
             <enum>Qt::ActionsContextMenu</enum>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="verticalScrollMode">
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
            <property name="horizontalScrollMode">
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
           </widget>
          </item>
          <item row="11" column="0">
           <widget class="QLabel" name="datis_info_label">
            <property name="text">
             <string>D-ATIS</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QTextBrowser" name="runway_info_display">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="runway_info_label">
            <property name="text">
             <string>Runways</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Runway Info</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tracks_information_tab">
       <attribute name="title">
        <string>Tracks Information</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QTabWidget" name="tracks_tabs">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="north_atlantic_tracks_tab">
             <attribute name="title">
              <string>North Atlantic Tracks</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="0" column="0">
               <widget class="QPushButton" name="get_north_atlantic_tracks_button">
                <property name="text">
                 <string>Fetch North Atlantic Tracks Data</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QTextBrowser" name="north_atlantic_text_display"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="pacific_tracks_tab">
             <attribute name="title">
              <string>Pacific Tracks</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="0" column="0">
               <widget class="QPushButton" name="get_pacific_tracks_button">
                <property name="text">
                 <string>Fetch Pacific Tracks Data</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QTextBrowser" name="pacific_tracks_display"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1455</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="toolbar_preferences_button"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="toolbar_preferences_button">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomQTableView</class>
   <extends>QTableView</extends>
   <header>zc_flightplan_toolkit.gui_classes</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
